### PR TITLE
Fix #150827

### LINKED ISSUE #150827: Deprecated extensions - warning message suggestions
Testing #150746

Old: 
This extension has been deprecated. Use Live Preview instead.
This extension is no longer being maintained and is deprecated.
This extension is deprecated and has become a native feature in VS Code. Configure these settings instead.

New:
This extension is deprecated. Install the Live Preview extension instead.
This extensions is deprecated as it is not longer being maintained.
This extension is deprecated as this functionality is now built-in to VS Code. Use these settings to enable the built-in functionality.

### PR DESCRIPTION
**Pull request by sandy081**
Fix #150827

### CODE REVIEW

File: src/vs/workbench/contrib/extensions/browser/extensionsActions.ts [hunk 1]
```diff
@@ -2295,12 +2295,12 @@ export class ExtensionStatusAction extends ExtensionAction {
...
 			} else if (this.extension.deprecationInfo.settings) {
 				const link = `[${localize('settings', "settings")}](${URI.parse(`command:workbench.action.openSettings?${encodeURIComponent(JSON.stringify([this.extension.deprecationInfo.settings.map(setting => `@id:${setting}`).join(' ')]))}`)})`;
-				this.updateStatus({ icon: warningIcon, message: new MarkdownString(localize('deprecated with alternate settings tooltip', "This extension is deprecated and has become a native feature in VS Code. Configure these {0} instead.", link)) }, true);
+				this.updateStatus({ icon: warningIcon, message: new MarkdownString(localize('deprecated with alternate settings tooltip', "This extension is deprecated as this functionality is now built-in to VS Code. Configure these {0} to use this functionality.", link)) }, true);
 			} else {
-				this.updateStatus({ icon: warningIcon, message: new MarkdownString(localize('deprecated tooltip', "This extension is no longer being maintained and is deprecated.")) }, true);
+				this.updateStatus({ icon: warningIcon, message: new MarkdownString(localize('deprecated tooltip', "This extensions is deprecated as it is no longer being maintained")) }, true);
```
Comment (lramos15) [inline src/vs/workbench/contrib/extensions/browser/extensionsActions.ts:2303]: Plural

**Review by lramos15 — APPROVED** [review]
Wording is a bit weird. Don't we normally stay way from using VS Code directly?

File: src/vs/workbench/contrib/extensions/browser/extensionsActions.ts [hunk 2]
```diff
@@ -2295,12 +2295,12 @@ export class ExtensionStatusAction extends ExtensionAction {
...
+				this.updateStatus({ icon: warningIcon, message: new MarkdownString(localize('deprecated with alternate extension tooltip', "This extension is deprecated. Use the {0} extension instead.", link)) }, true);
 			} else if (this.extension.deprecationInfo.settings) {
 				const link = `[${localize('settings', "settings")}](${URI.parse(`command:workbench.action.openSettings?${encodeURIComponent(JSON.stringify([this.extension.deprecationInfo.settings.map(setting => `@id:${setting}`).join(' ')]))}`)})`;
-				this.updateStatus({ icon: warningIcon, message: new MarkdownString(localize('deprecated with alternate settings tooltip', "This extension is deprecated and has become a native feature in VS Code. Configure these {0} instead.", link)) }, true);
+				this.updateStatus({ icon: warningIcon, message: new MarkdownString(localize('deprecated with alternate settings tooltip', "This extension is deprecated as this functionality is now built-in to VS Code. Configure these {0} to use this functionality.", link)) }, true);
```
[the remark this answers is not in the document]
Comment (sandy081) [reply src/vs/workbench/contrib/extensions/browser/extensionsActions.ts:2301]: Good point. But we use VS Code as a default general product name in many places. Followed the same

File: src/vs/workbench/contrib/extensions/browser/extensionsActions.ts (hunk 1 above)
Comment (sandy081) [reply src/vs/workbench/contrib/extensions/browser/extensionsActions.ts:2303]: Good catch

**Pull request merged**